app/components/recirculation-prefooter.js

```diff
@@ -24,6 +24,40 @@ export default Component.extend(
 		isVisible: false,
 		liftigniter: inject.service(),
 		i18n: inject.service(),
+		hasNoLiftigniterSponsoredItem: true,
+		isInRightCountry: false,
+		shouldShowPlista: computed.and('hasNoLiftigniterSponsoredItem', 'isInRightCountry'),
```

Comment:
shouldShowPlista: comupted(function() {
 return ['AU', 'NZ'].indexOf(M.Geo....) > -1 * this.get('hasNo...')
})